src/synth/fluid_gen.c

```diff
@@ -25,70 +25,70 @@
 
 /* See SFSpec21 $8.1.3 */
 static const fluid_gen_info_t fluid_gen_info[] = {
-        /* number/name             init  scale         min        max         def */
```

Comment:
Is there a way to protect these this nicely formatted code from clang-format?

Reply:
Only via explicit `/* clang-format off */`, or if we only used astyle, see #404.